Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
        orig_or = re.RegexFlag.__or__
        def mock_or(self, other):
            if hasattr(other, "orig"):
                return self | other.orig
            else:
                return orig_or(self, other)
        re.RegexFlag.__or__ = mock_or
    
        re.MULTILINE = MockReFlag(re.MULTILINE)
        re.IGNORECASE = MockReFlag(re.IGNORECASE)
>       from string_utils import _regex

../test.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
string_utils/__init__.py:7: in <module>
    from .validation import *
string_utils/validation.py:35: in <module>
    from ._regex import *
string_utils/_regex.py:153: in <module>
    NO_LETTERS_OR_NUMBERS_RE = re.compile(r'[^\w\d]+|_+', re.IGNORECASE + re.UNICODE)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <test.MockReFlag object at 0x7d096e5b0160>, other = re.UNICODE

    def __add__(self, other):
>       assert False
E       assert False

../test.py:8: AssertionError
=========================== short test summary info ============================
FAILED ../test.py::test - assert False
============================== 1 failed in 0.05s ===============================
